glade_forms/lib/src/validator/string_validator.dart

```diff
@@ -17,6 +17,19 @@ class StringValidator extends GladeValidator<String> {
         shouldValidate: shouldValidate,
       );
 
+  /// Given value can't be blank.
+  void notBlank({
+    OnValidateError<String>? devError,
+    Object? key,
+    ShouldValidateCallback<String>? shouldValidate,
+  }) =>
+      satisfy(
```

Comment:
Thinking about that we could extend notEmpty validator with boolean "includeBlank"  rather than adding new one

Reply:
@tenhobi  what do you think?

Reply:
in our flutter_core we have a lot of extensions with blank already https://github.com/netglade/flutter_core/blob/main/packages/netglade_utils/lib/src/extensions/string_extensions.dart#L15

That does not necessary create some precedence tho for keeping it here as well. Not sure what is better, both are probably ok.

we can have notEmpty with some parameter, however `includeBlank` is probably not ideal name, since the "not empty" with "include blank" is confusing. Rather something like "not empty" with "ignore blank (characters)" / "without blanks" or something like that?

Reply:
Yeah ignoreBlank is quite better but still little bit confusing, setting ignoreBlank:true means that blank are allowed or opposite?

notEmpty(ignoreBlank: true) can be read as "String should be not empty but ignore blank characters" - thus blank characters are allowed **OR** it can be read as "String should be not empty and ignore blank characters"... so 😅 

Any other names?

Reply:
Pfff. :D What about allowBlank (default true)? I am not sure anymore whether adding another method is not simpler than the parameter haha.

`notEmpty(allowBlank: true)`
- `` - false
- ` ` - true
- `a` - true

`notEmpty(allowBlank: false)` (aka "notBlank" method)
- `` - false
- ` ` - false
- `a` - true

Reply:
I am for allowBlank